governance/policies/archiving.md

```diff
@@ -0,0 +1,102 @@
+# opam Repository Archiving Policy
+
+*NOTE*: This policy is not static and will be updated as the process matures.
```

Comment:
This is going to age quickly. I suggest defining some dates in the future at which point the process is either ratified or discarded. This is similar to how (e.g.) IETF drafts expire if they are not standardised.

Reply:
We will have dates very soon in the announcement for the implementation. Then we can update this by saying that the archival policy will be consolidated by that day but that we reserve the rights to modify it to fit with the evolution of the repository. Would that be a reasonable compromise?

Reply:
I had the impression that we wanted this note to be perpetual, as a general warning that policies are not static, and need updating as circumstances and processes change.

That said, IMO, this is implicit a notice to that effect is not really necessary, so I am not opposed to to adding a deadline or just removing this entirely.

Reply:
Let's remove it altogether then